### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by fponticelli** [FuzzyWuzzie/gjk-demos#1]
Just wanted to say that I am really liking how you used MiniCanvas for this project. I feel like you nailed a really powerful use case and I wonder if there is anything major that can be done to improve its usage.
Thanks!

**Comment by hamaluik** [comment]
Thanks for making Minicanvas! I really like using it for things like this.

I think my only real issue is the animation. I started trying to do more interesting animation stuff, but got confused rather quickly so dropped back to using an async library to periodically clear the canvas, rebuild it, etc. I'm not sure I have an exact recommendation here as I don't remember exactly what the struggle was, just that I started going down a rabbit hole with it and decided to scrap it in order to get things working rather than looking good in the time I gave myself for it.

Oh, also text. Doesn't seem like Minicanvas has anything to do with text, so I had to grab the underlying context and use native functions to draw text (which is fine, but was the only thing I found missing).
